zuul.d/jobs.yaml

```diff
@@ -84,3 +84,24 @@
     voting: false
     vars:
       tox_envlist: molecule-aio-ovn
+
+- job:
+    name: atmosphere-molecule-aio-upgrade-base
+    run: zuul.d/playbooks/upgrade/run-upgrade.yml
+    abstract: true
+    timeout: 10800
+    parent: atmosphere-molecule-aio-full
+    vars:
+      from_branch: "stable/2023.2"
+
+- job:
+    name: atmosphere-molecule-aio-upgrade-ovs
```

Comment:
For consistency, please rename the job to use `openvswitch` instead of `ovs`.

Reply:
done